Kajabi/sage-lib
# Sage3 Hero Updates

## Description
This PR updates the Sage Hero component to match the Sage 3 specs.

## Screenshots
|  Before  |  After  |
|--------|--------|
|<img width="1263" alt="Before" src="https://user-images.githubusercontent.com/7331038/133479074-9ff52399-b897-47b1-82e0-ecbdb4c09b43.png">|<img width="1137" alt="After" src="https://user-images.githubusercontent.com/7331038/133479373-cd688c2d-1123-4277-b4d5-a122ecf63c06.png">|

## Testing in `sage-lib`
1. Visit http://0.0.0.0:4000/pages/component/hero
2. Ensure that the `small` and `large` sizes match the Figma specs

## Testing in `kajabi-products`
1. (**LOW**) SageHero is not currently used.
